service/src/main/resources/WEB-INF/templates/tampere/shared/daycare-correction.html

```diff
@@ -10,60 +10,23 @@
 <body>
 <th:block th:fragment="correction">
     <h1 th:text="#{title}"></h1>
-    <div th:if="${#locale.toString()} == 'fi'" class="correction-content">
-        <p th:text="#{section.main.1}"></p>
-        <p th:text="#{section.main.2}"></p>
-        <p th:text="#{section.main.3}"></p>
-        <p th:text="#{section.main.4}"></p>
-        <ul>
-            <li th:text="#{section.main.4.list.1}"></li>
-            <li th:text="#{section.main.4.list.2}"></li>
-            <li th:text="#{section.main.4.list.3}"></li>
-            <li th:text="#{section.main.4.list.4}"></li>
-        </ul>
-        <p th:text="#{section.main.5}"></p>
-        <p th:text="#{section.main.6}"></p>
+    <div class="correction-content">
+        <span th:utext="#{content}"></span>
```

Comment:
p element should not be inside a span element. Better to use p and remove it from the properties, or wrap the p in a div. Check other places also.